src/code/graph.c

```diff
@@ -119,7 +126,7 @@ void Graph_InitTHGA(GraphicsContext* gfxCtx) {
     gfxCtx->unk_014 = 0;
 }
 
-GameStateOverlay* Graph_GetNextGameState(GameState* gameState) {
+struct GameStateOverlay* Graph_GetNextGameState(GameState* gameState) {
```

Comment:
```suggestion
GameStateOverlay* Graph_GetNextGameState(GameState* gameState) {
```

Not required since you are including `z_game_dlftbls.h`